app/_how-tos/kic-get-started-proxy-caching.md

```diff
@@ -0,0 +1,123 @@
+
+## Proxy Cache Headers
+The `proxy-cache` plugin returns a `X-Cache-Status` header that can contain the following cache results:
+
+{% table %}
+columns:
+  - title: State
+    key: state
+  - title: Description
+    key: desc
+rows:
+  - state: Miss
+    desc: |
+      The request could be satisfied in cache, but an entry for the resource was not found in cache, and the request was proxied upstream.
+  - state: Hit
+    desc: |
+      The request was satisfied and served from the cache.
+  - state: Refresh
+    desc: |
+      The resource was found in cache, but could not satisfy the request, due to Cache-Control behaviors or reaching its hard-coded `cache_ttl` threshold.
+  - state: Bypass
+    desc: |
+      The request could not be satisfied from cache based on plugin configuration
+{% endtable %}
+
+## Create a proxy-cache KongClusterPlugin
+
+In the previous section you created a `KongPlugin` that was applied to a specific service or route. You can also use a `KongClusterPlugin` which is a global plugin that applies to all services.
+
+This configuration caches all `HTTP 200` responses to `GET` and `HEAD` requests for 300 seconds:
+
+{% entity_example %}
+type: plugin
+data:
+  name: proxy-cache-all-endpoints
+  plugin: rate-limiting
+  config:
+    response_code:
+    - 200
+    request_method:
+    - GET
+    - HEAD
+    content_type:
+    - text/plain; charset=utf-8
+    cache_ttl: 300
+    strategy: memory
+{% endentity_example %}
+
+## Test the proxy-cache plugin
+
+To test the proxy-cache plugin, send another six requests to `$PROXY_IP/echo`:
+
+{% validation rate-limit-check %}
+iterations: 6
+url: '/echo'
+headers:
+  - 'apikey:example-key'
+on_prem_url: $PROXY_IP
+konnect_url: $PROXY_IP
+grep: "(Status|< HTTP)"
+message: null
+output:
+  explanation: |
+    The first request results in `X-Cache-Status: Miss`. This means that the request is sent to the upstream service. The next four responses return `X-Cache-Status: Hit` which indicates that the request was served from a cache.
+  expected:
+    - value:
+      - "< HTTP/1.1 200 OK"
+      - "< X-Cache-Status: Hit"
+    - value:
+      - "< HTTP/1.1 200 OK"
+      - "< X-Cache-Status: Hit"
+    - value:
+      - "< HTTP/1.1 200 OK"
+      - "< X-Cache-Status: Hit"
+    - value:
+      - "< HTTP/1.1 200 OK"
+      - "< X-Cache-Status: Hit"
+    - value:
+      - "< HTTP/1.1 200 OK"
+      - "< X-Cache-Status: Hit"
+    - value:
+      - "< HTTP/1.1 429 Too Many Requests"
+{% endvalidation %}
+
+The final thing to note is that when a `HTTP 429` request is returned by the rate-limit plugin, you do not see a `X-Cache-Status` header. This is because `rate-limiting` executes before `proxy-cache`. For more information, see [plugin priority](/gateway/entities/plugin/#plugin-priority).
```

Comment:
```suggestion
The final thing to note is that when a `HTTP 429` request is returned by the rate-limit plugin, you don't see a `X-Cache-Status` header. This is because `rate-limiting` executes before `proxy-cache`. For more information, see [plugin priority](/gateway/entities/plugin/#plugin-priority).
```